packages/connectivity_plus/connectivity_plus_linux/pubspec.yaml

```diff
@@ -18,9 +18,10 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  build_runner: ^1.11.5
-  mockito: ^5.0.0
-  flutter_lints: ^1.0.4
+  build_runner: ^2.1.11
+  dbus: ^0.7.5
```

Comment:
Had to add as otherwise `flutter pub publish --dry-run` wouldn't find Dbus even though it is  a direct dependency of `nm`. 
https://github.com/canonical/nm.dart/blob/main/pubspec.yaml#L16